thrimbletrimmer/scripts/edit.js

```diff
@@ -668,7 +672,11 @@ async function initializeVideoInfo() {
 	if (videoInfo.video_description) {
```

Comment:
If `video_description` is the empty string (ie. someone deleted the description then saved draft) we don't want to use the default. We only want that when `video_description` is `null`:
```suggestion
	if (videoInfo.video_description === null) {
```

Reply:
With how the logic is currently organized, that should be `!== null`, right?